compiler/plugins/target/AMD-AIE/aie/test/link_test_join.mlir

```diff
@@ -1,5 +1,6 @@
 
 // RUN: iree-opt --amdaie-objectFifo-stateful-transform %s | FileCheck %s
+// XFAIL: *
```

Comment:
What's erroneous about this test (& others)? The external_buffer?

Reply:
yes (and/or `aie.shim_dma`)